src/components/picker/index.js

```diff
@@ -128,6 +128,7 @@ class Picker extends BaseComponent {
 
   static defaultProps = {
     ...TextField.defaultProps,
+    enableErrors: undefined,
```

Comment:
if anything define the default to be false, all props by default are undefined.

Reply:
Removed